tests/test_fixup_scripts.py

```diff
@@ -187,3 +190,28 @@ def test_fixup_script_different_version(self):
         assert clone_path in data
 
         assert clone_path + '/bin/python2' in data
+
+    def test_fixup_pth_file(self):
+        '''Prior to version 0.2.7, the method fixup_pth_file() wrongly wrote unicode files'''
+
+        content = os.linesep.join(('/usr/local/bin/someFile', '/tmp/xyzzy', '/usr/local/someOtherFile', ''))
+        pth = '/tmp/test_fixup_pth_file.pth'
+
+        with open(pth, 'w') as fd:
+            fd.write(content)
+
+        # Find size of file - should be plain ASCII/UTF-8...
+
+        orgStat = os.stat(pth)
+        assert len(content) == orgStat[stat.ST_SIZE]
+
+        # Now sleep for 2 seconds, then call fixup_pth_file(). This should ensure that the stat.ST_MTIME has
+        # changed if the file has been changed/rewritten
+
+        time.sleep(2)
```

Comment:
Can you instead either set a timestamp in the past or mock the `os.stat` call to provide a later?  A 2-second sleep makes the tests slower, so I'd like to avoid adding it to the test suite..

Reply:
The test suite already runs for 58 seconds. I would argue that those 2 extra seconds is well worth it for simplicity rather than adding of some obscure fiddling with timestamps or mocking. KISS :-)